### PR TITLE
List albums / play album / play different album implementation 

### PR DESCRIPTION
**Pull request by Danondso**
Implemented:
Play latest album
List all albums
List latest albums (up to five)
Play album by artist
Play different album

If someone could test out the changes that would be great. Let me know if anything should be changed!

### CODE REVIEW

File: geemusic/utils/music.py [hunk 1]
```diff
@@ -75,6 +76,42 @@ def get_album(self, name, artist_name=None):
...
+            return False
+
+        artist_info = self._api.get_artist_info(search[0]['artistId'], include_albums=True)
+        album_info = artist_info['albums']
+        random.shuffle(album_info)
+
+        for index, val in enumerate(album_info):
+            album = self._api.get_album_info(album_id=album_info[index]['albumId'], include_tracks=True)
+            if album['albumId'] != album_id:
+                if len(album['tracks']) >= 5:
```
Comment (stevenleeg) [inline geemusic/utils/music.py]: any reason this isn't using an and?
```python
if album['albumId'] != album_id and len(album['tracks']) >=5:
```

Comment (Danondso) [inline geemusic/utils/music.py]: No reason lol, just an oversight on my part.

**Comment by Danondso** [comment]
Is there anything else you need me to change on this?

**Comment by stevenleeg** [comment]
nope, sorry for the delay and mega thanks for the contribution!

**Comment by Danondso** [comment]
Yeah you're welcome!

**Pull request merged**